src/services/completions.ts

```diff
@@ -1651,7 +1651,8 @@ namespace ts.Completions {
             if (!isClassLike(decl)) return GlobalsSearch.Success;
 
             const classElement = contextToken.parent;
-            let classElementModifierFlags = isClassElement(classElement) ? getModifierFlags(classElement) : ModifierFlags.None;
+            // If the context token is semi token, we can think that it is already after the last entry
+            let classElementModifierFlags = contextToken.kind !== SyntaxKind.SemicolonToken && isClassElement(classElement) ? getModifierFlags(classElement) : ModifierFlags.None;

```

Comment:
`classElement` *should* be a `SemicolonClassElement`. I think a better idea would be to make `isClassElement` return `true` for `SemicolonClassElement`s

Reply:
why `classElement` should be a `SemicolonClassElement`?.. that actually a `PropertyDeclaration`  

is that mean we should extract the semi as `SemicolonClassElement` rather than a part of `PropertyDeclaration`?

Reply:
I'm also a little confused on this fix. It seems like the actual issue might be that `tryGetObjectTypeDeclarationCompletionContainer` doesn't include `;` in its `switch` block?

Why doesn't this bug manifest when the `contextToken` is the closing `}` of a method above the caret?

Reply:
> Why doesn't this bug manifest when the `contextToken` is the closing `}` of a method above the caret?

@RyanCavanaugh I assume because the parent of the `}` is a `MethodDeclaration` which is considered a `ClassElement`

> is that mean we should extract the semi as `SemicolonClassElement` rather than a part of `PropertyDeclaration`?

Good point - I would have imagined that a `PropertyDeclaration` should be accounted for in `isClassElement`.

Reply:
@DanielRosenwasser 
>  I think a better idea would be to make isClassElement return true for SemicolonClassElements

that already is

@RyanCavanaugh 
> It seems like the actual issue might be that tryGetObjectTypeDeclarationCompletionContainer doesn't include ; in its switch block?  

seems that already is too🤷🏻‍♂️